chatmemorystore-redis/deployment/src/main/java/io/quarkiverse/langchain4j/chatmemorystore/redis/deployment/RedisChatMemoryStoreBuildTimeConfig.java

```diff
@@ -0,0 +1,19 @@
+package io.quarkiverse.langchain4j.chatmemorystore.redis.deployment;
+
+import static io.quarkus.runtime.annotations.ConfigPhase.BUILD_TIME;
+
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
+
+@ConfigRoot(phase = BUILD_TIME)
+@ConfigMapping(prefix = "quarkus.langchain4j.chatmemorystore.redis")
```

Comment:
@jmartisk any suggestions on improving this name? Sounds like a little to much for users to type...